vitrina/orgs/models.py

```diff
@@ -82,20 +82,32 @@ def get_absolute_url(self):
 
 
 class Representative(models.Model):
+    COORDINATOR = 'coordinator'
+    MANAGER = 'manager'
+    ROLES = {
+        (COORDINATOR, _("Koordinatorius")),
+        (MANAGER, _("Tvarkytojas"))
+    }
+
     created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     modified = models.DateTimeField(blank=True, null=True, auto_now=True)
-    version = models.IntegerField()
+    version = models.IntegerField(default=1)
     email = models.CharField(max_length=255, blank=True, null=True)
     first_name = models.CharField(max_length=255, blank=True, null=True)
     last_name = models.CharField(max_length=255, blank=True, null=True)
     organization = models.ForeignKey(Organization, models.DO_NOTHING, blank=True, null=True)
```

Comment:
Never use `DO_NOTHING` this will cause integrity errors and basically corrupts database. Also this field can't be NULL nor blank, because it participates in unique together constraint.